.travis.yml

```diff
@@ -1,10 +1,15 @@
 language: php
+
 php:
   - 7.1
   - 7.2
-before_script:
+
+before_install:
   - composer self-update
```

Comment:
This is not needed, travis has this included.

Reply:
See https://github.com/phpstan/phpstan-strict-rules/pull/72.